styles.css

```diff
@@ -439,3 +439,13 @@ body.dark .footer::before {
         margin-bottom: 10%;
     }
 }
+
+/* --------------------------------------------------------ADDED A TRANSFORM EFFECT--------------------------------------------- */
+.effect:hover{
+    font-size: 1.2rem;
+    color: rgb(206, 14, 240);
+}
+#effect:hover{
+    color: red;
+    font-size: 1.2rem;
```

Comment:
I think different color would be nice to have for different theme. Try using `body.dark` and `body.light` directives before `.effect` and `#effect` to set colors for different themes. Try choosing some aesthetic colors, nothing too eye-blinding such as red ;).